migrations/20210823233133_create_team_table.sql

```diff
@@ -0,0 +1,15 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE team(
+    id  SERIAL PRIMARY KEY,
+    name VARCHAR(100) NOT NULL,
+    description VARCHAR(255) NOT NULL
+)
+-- +goose StatementEnd
+
+
+
+-- +goose Down
+-- +goose StatementBegin
+DROP TABLE IF EXISTS team
```

Comment:
Если оно падает – оно должно упасть, чтобы мы сразу об этом узнали. Мы не должны здесь подстеливать солому, потому что может произойти ошибка, о которой мы узнаем слишком поздно – или вообще не узнаем
```suggestion
DROP TABLE team
```

Reply:
понял, поправил